Prace_domowe/Praca_domowa2/Jamtozy_Bartosz/pd2_allegro.ipynb

```diff
@@ -0,0 +1,2536 @@
+{
```

Comment:
<p>Błąd należałoby wyliczyć tylko dla wartości imputowanych, bo gdy wyliczamy dla całej kolumny wynik jest zakłamany, ponieważ we wzorze jest wyliczana średnia, przez co liczność ma znaczenie. </p><p> </p><p>Prośba o zmodyfikowanie (odnosi się to również do analiz poniżej)</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/67/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>